R/util.R

```diff
@@ -1439,11 +1439,6 @@ extract_from_date <- function(x, type = "fltoyear") {
     wdaylong = {
       ret <- lubridate::wday(x, label=TRUE, abbr=FALSE)
     },
-    wdaytype = {
```

Comment:
Don't you need to keep wdaytype option here to make chart etc. keep working?

Reply:
Chart/Analytics/Summarize dialog use `weekend` instead of `extact_from_date` function so it works.

Reply:
Bring back weekend.